bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccBindingConstants.java

```diff
@@ -110,20 +113,14 @@ public class EvccBindingConstants {
             CHANNEL_LOADPOINT_MAX_CURRENT);
     public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_MIN_CURRENT = new ChannelTypeUID(BINDING_ID,
             CHANNEL_LOADPOINT_MIN_CURRENT);
-    public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_MIN_SOC = new ChannelTypeUID(BINDING_ID,
-            CHANNEL_LOADPOINT_MIN_SOC);
     public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_MODE = new ChannelTypeUID(BINDING_ID,
             CHANNEL_LOADPOINT_MODE);
     public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_PHASES = new ChannelTypeUID(BINDING_ID,
             CHANNEL_LOADPOINT_PHASES);
-    public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_TARGET_ENERGY = new ChannelTypeUID(BINDING_ID,
-            CHANNEL_LOADPOINT_TARGET_ENERGY);
-    public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_TARGET_SOC = new ChannelTypeUID(BINDING_ID,
-            CHANNEL_LOADPOINT_TARGET_SOC);
-    public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_TARGET_TIME = new ChannelTypeUID(BINDING_ID,
-            CHANNEL_LOADPOINT_TARGET_TIME);
-    public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_TARGET_TIME_ENABLED = new ChannelTypeUID(BINDING_ID,
-            CHANNEL_LOADPOINT_TARGET_TIME_ENABLED);
+    public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_LIMIT_ENERGY = new ChannelTypeUID(BINDING_ID,
+            CHANNEL_LOADPOINT_LIMIT_ENERGY);
+    public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_LIMIT_SOC = new ChannelTypeUID(BINDING_ID,
+            CHANNEL_LOADPOINT_LIMIT_SOC);
     public static final ChannelTypeUID CHANNEL_TYPE_UID_LOADPOINT_TITLE = new ChannelTypeUID(BINDING_ID,
             CHANNEL_LOADPOINT_TITLE);
```

Comment:
But you of course need to use the new channel type:

```suggestion
            "loadpointVehicleTitle");
```

Reply:
I think this is wrong. LOADPOINT_TITLE describes the title of the loadpoint and the loadpointVehicleTitle is the title of the vehicle that is currently connected to the loadpoint. I solved you suggestion above (renaming to vehicleTitle for CHANNEL_LOADPOINT_VEHICLE_TITLE and so in thing-types.xml)

Reply:
> I think this is wrong.

Oops sorry, thx for solving my suggestion.

BTW: Please don‘t mark review suggestions as resolved once you resolved them, it‘s up to the reviewer to mark them as resolved. I usually use an reaction emoji to mark for myself what I already resolved when addressing an review.

Reply:
> BTW: Please don‘t mark review suggestions as resolved once you resolved them, it‘s up to the reviewer to mark them as resolved. I usually use an reaction emoji to mark for myself what I already resolved when addressing an review.

oh yeah ofc sorry. First time contributing to a project like this :)